client/app/components/dashboards/AddWidgetDialog.jsx

```diff
@@ -209,6 +210,8 @@ class AddWidgetDialog extends React.Component {
                     onClick={() => this.selectQuery(query.id)}
                   >
                     {query.name}
+                    {' '}
+                    <QueryTagsControl tags={query.tags} className="label" />
```

Comment:
`className="inline-tags-control"` (and also in other places)

Reply:
Thanks! I've updated to use this as well. I'm not the most savvy with css, but I did my best to ensure that alignment is looking accurate, needed to change some existing css, but I think it didn't have any negative effects. Let me know if I've missed something!

Reply:
Thanks @emtwo! You can try `vertical-align: middle` and `vertical-align: baseline` CSS and choose what looks better.